grudge/dof_desc.py

```diff
@@ -96,29 +98,44 @@ def __repr__(self):
         return "<{}({})>".format(type(self).__name__, repr(self.tag))
 
 
-class QTAG_NONE:  # noqa: N801
-    """A quadrature tag indicating the use of a
+class DISCR_TAG_BASE:  # noqa: N801
+    """A discretization tag indicating the use of a
     basic discretization grid. This tag is used
-    to distinguish the base discretization (`QTAG_NONE`)
-    from quadrature (e.g. overintegration) grids.
+    to distinguish the base discretization from quadrature
+    (e.g. overintegration) or modal (`DISCR_TAG_MODAL`)
+    discretizations.
+    """
+
+
+# FIXME: This needs to go away
+class QTAG_NONE:  # noqa: N801
+    pass
+
+
+class DISCR_TAG_QUAD:  # noqa: N801
+    """A discretization tag indicating the use of a
+    quadrature discretization grid. This tag is used
+    to distinguish the quadrature discretization
+    (e.g. overintegration) from modal (`DISCR_TAG_MODAL`)
+    or base (`DISCR_TAG_BASE`) discretizations.
     """
 
 
-class QTAG_MODAL:  # noqa: N801
-    """A quadrature tag indicating the use of a
+class DISCR_TAG_MODAL:  # noqa: N801
+    """A discretization tag indicating the use of a
     basic discretization grid with modal degrees of
     freedom. This tag is used to distinguish the
-    modal discretization (`QTAG_MODAL`) from
-    the base (nodal) discretization (e.g. `QTAG_NONE`)
-    or discretizations on quadrature grids.
+    modal discretization from the base (nodal)
+    discretization (e.g. `DISCR_TAG_BASE`) or
+    discretizations on quadrature grids (`DISCR_TAG_QUAD`).
```

Comment:
```suggestion
    discretization (e.g. :class:`DISCR_TAG_BASE`) or
    discretizations on quadrature grids (:class:`DISCR_TAG_QUAD`).
```

Reply:
Done in https://github.com/inducer/grudge/pull/83/commits/4ed7e6f15401b6cb5da85c84f45ec267dbc2db1f